web/views/permissions_view.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Cog.V1.PermissionView do
+  use Cog.Web, :view
+
+  def render("show.json", %{permission: permission}) do
+    %{permission: %{id: permission.id,
+                    name: permission.name,
+                    namespace: permission.namespace.name} }
+  end
+
+  def render("index.json", %{permissions: permissions}) do
+    %{permissions: Enum.map(permissions, fn(permission) ->
+        %{id: permission.id,
+          name: permission.name,
+          namespace: permission.namespace.name}
+    end)}
+  end
```

Comment:
Nice! This looks really nice.

It's probably not necessary for this PR but one nice thing I've seen people do is add a `render` function head for `permission.json` and then use that in all of the action specific functions (`index`, `show`, etc), so that the logic is the same.